src/main/java/world/bentobox/bentobox/listeners/JoinLeaveListener.java

```diff
@@ -197,8 +198,22 @@ private void updateIslandRange(User user) {
                     user.sendMessage("commands.admin.setrange.range-updated", TextVariables.NUMBER, String.valueOf(range));
                     plugin.log("Island protection range changed from " + island.getProtectionRange() + " to "
                             + range + " for " + user.getName() + " due to permission.");
+
+                    // Get old range for event
+                    int oldRange = island.getProtectionRange();
+
+                    island.setProtectionRange(range);
+
+                    // Call Protection Range Change event. Does not support cancelling.
+                    IslandEvent.builder()
+                        .island(island)
+                        .location(island.getCenter())
+                        .reason(IslandEvent.Reason.RANGE_CHANGE)
+                        .involvedPlayer(user.getUniqueId())
+                        .admin(true)
```

Comment:
Are you sure admin should be set to true here and in the IslandsManager? I personally can't recall what this admin value is used for.

Reply:
Yea, I just copy-paste it as in all other IslandEvents so, it contains both, admin and location, even if it is not necessary for this particular event.

Reply:
Could you please get rid of these useless statements then?

Reply:
I checked the usage of location and admin, and they are used in IslandBaseEvent calling.
I could remove them from passing, but then this IslandBaseEvent will be called with admin = false and location = null in line: https://github.com/BentoBoxWorld/BentoBox/blob/develop/src/main/java/world/bentobox/bentobox/api/events/island/IslandEvent.java#L443

Reply:
Okay, so location is expected to be the island center. Let's keep admin as true for now. We will see later on if that causes issues.